File: tools/bench/answers/go_sonnet/s3/12_celsius.go
package main

import "fmt"

func main() {
	temps := []float64{12.5, 30.0, -5.0}
	maxF := temps[0]*9/5 + 32
	for _, c := range temps[1:] {
		f := c*9/5 + 32
		if f > maxF {
			maxF = f
		}
	}
	fmt.Println(maxF)
}

File: tools/bench/answers/rust_haiku/s1/12_celsius.rs
fn main() {
    let temps = [12.5, 30.0, -5.0];

    let mut max_f = (temps[0] * 9.0 / 5.0) + 32.0;

    for temp in temps.iter() {
        let f = (temp * 9.0 / 5.0) + 32.0;
        if f > max_f {
            max_f = f;
        }
    }

    println!("{}", max_f);
}
